frontend/src/pages/Batch.tsx

```diff
@@ -90,11 +86,70 @@ const Batch: React.FC = () => {
     );
   }, [batch]);
 
+  function getSparkSubmitArg(key: string, value: string) {
+    if (value) {
+      return ' ' + key + ' ' + value;
+    } else {
+      return '';
+    }
+  }
+
+  const sparkSubmitStr = useMemo(() => {
+    if (!batch) {
+      return null;
+    }
+
+    return (
+      <Code className={styles.logs}>
+        spark-submit
+        {getSparkSubmitArg('--name', batch.submitParams.name)}
+        {getSparkSubmitArg('--driver-cores', batch.submitParams.driverCores.toString())}
+        {getSparkSubmitArg('--driver-memory', batch.submitParams.driverMemory)}
+        {getSparkSubmitArg('--num-executors', batch.submitParams.numExecutors.toString())}
+        {getSparkSubmitArg('--executor-cores', batch.submitParams.executorCores.toString())}
+        {getSparkSubmitArg('--executor-memory', batch.submitParams.executorMemory)}
+        {getSparkSubmitArg('--py-files', batch.submitParams.pyFiles.join(','))}
+        {getSparkSubmitArg('--archives', batch.submitParams.archives.join(','))}
+        {getSparkSubmitArg('--files', batch.submitParams.files.join(','))}
+        {getSparkSubmitArg('--jars', batch.submitParams.jars.join(','))}
+        {Object.entries(batch.submitParams.conf).map(([name, val]) => (
+          <span key={name}>
+            --conf {name} {val + ' '}
```

Comment:
why do you need whitespace at the end?

Reply:
Without white space it's like this: `--conf name1 val1--conf nam2 val2--conf...`. Maybe there is a better way to join all values with a space.